### PR TITLE
Add debug option to the sheet printer plugin

### PR DESCRIPTION
**Pull request by martonmiklos**
I spent some time debugging an issue with my labels printed through the sheet printer (looks to be a wheasyprint issue).
At some point I went ahead and added a debug settings to being able to see the raw HTML output. It might be useful to others in the future.

### CODE REVIEW

**Comment by SchrodingersGat** [comment]
@martonmiklos great addition, thanks!

File: src/backend/InvenTree/plugin/builtin/labels/label_sheet.py [hunk 1]
```diff
@@ -64,9 +65,22 @@ class InvenTreeLabelSheetPlugin(LabelPrintingMixin, SettingsMixin, InvenTreePlug
...
+        }
+    }
 
     PrintingOptionsSerializer = LabelPrintingOptionsSerializer
+    debug = None
+
+    def in_debug_mode(self):
+        """Check if the plugin is printing in debug mode."""
+        if self.debug is None:
+            self.debug = str2bool(self.get_setting('DEBUG'))
```
Comment (SchrodingersGat) [inline src/backend/InvenTree/plugin/builtin/labels/label_sheet.py:83]: Oops, this function should return a value, right?

**Comment by SchrodingersGat** [comment]
@martonmiklos I think I merged this too hastily - please see comment above. The `in_debug_mode` method should return a value. Please submit a new PR to address this :)

File: src/backend/InvenTree/plugin/builtin/labels/label_sheet.py (hunk 1 above)
Comment (wolflu05) [reply src/backend/InvenTree/plugin/builtin/labels/label_sheet.py:83]: Also, what's the purpose of setting self.debug if it's not defined? The plugin settings are cached. And secondly this means that we cannot disable debug mode at runtime anymore.

File: src/backend/InvenTree/plugin/builtin/labels/label_sheet.py [hunk 2]
```diff
@@ -135,11 +149,15 @@ def print_labels(
...
-        html = weasyprint.HTML(string=html_data)
-        document = html.render().write_pdf()
+        if self.in_debug_mode():
+            # Render HTML to PDF
+            html = weasyprint.HTML(string=html_data)
+            document = html.render().write_pdf()
+
+            output.output = ContentFile(document, 'labels.pdf')
+        else:
+            output.output = ContentFile(html_data, 'labels.html')
```
Comment (wolflu05) [inline src/backend/InvenTree/plugin/builtin/labels/label_sheet.py:159]: Isn't that just swapped? I mean if in debug mode it should return the html, but you return the pdf.

Comment (SchrodingersGat) [reply src/backend/InvenTree/plugin/builtin/labels/label_sheet.py:159]: @wolflu05 correct, I was way too hasty on this. @martonmiklos please review

Comment (martonmiklos) [reply src/backend/InvenTree/plugin/builtin/labels/label_sheet.py:159]: Ah, copy paste engineering tricked me, let me fix it.

**Pull request merged**